gravityflow/gravityflowblocks
# Fix several componentWillUnmount methods are accidentally removed

This PR fixes several `componentWillUnmount` methods are accidentally removed from the Edit classes.

Based on the last update [here](https://github.com/WordPress/gutenberg/issues/5626#issuecomment-612274055), such a hack is still needed for blocks using post meta.

## Testing instructions
1. Adding a Gravityflow Inbox block to your page, setting up "Filter Forms" attribute and save the page.
2. Go to your database and find the post meta `_gravityflow_inbox_forms_json` is there with a value.
3. Delete the Inbox block, with this branch, when you check the  the `_gravityflow_inbox_forms_json` value, it will be empty.
